test/functional/inspec_exec_test.rb

```diff
@@ -1002,6 +1002,30 @@ def stderr
       end
     end
 
+    describe "when using the --config option and --reporter option to configure reporter in a run in a correct manner" do
+      outpath = Dir.tmpdir
+      let(:cli_args) { "--config " + File.join(config_dir_path, "json-config", "reporter-json-config.json") + " --reporter json:#{outpath}/test-json-report.json" }
```

Comment:
So, this isn't a very clear test to me. I want a test that says, clearly, the --config will create THIS file, and I will check for it; and the --reporter option will create THAT file, and I will check for it. 

Why are we expecting anything at all on STDOUT?

Reply:
I have forgotten to commit and push the config file that I created for this test, so that would have cleared up why I am testing `stdout`.
So the above tests intend to test the merging of options passed using reporter CLI and using config.
So if the config file says STDOUT: true, then it should obey that.
Yes, the best test would be with STDOUT false and the file path, but testing that by providing a dynamic path in config file seemed difficult. Let me try modifying these tests.